CDTDatastore/CDTReplicator/CDTReplicator.m

```diff
@@ -50,7 +50,33 @@ @interface CDTReplicator ()
 @property (nonatomic) BOOL started;
 
 @end
-
+/*
+    The replicatore supports "Fire and forget" this is where the application releases it's reference
```

Comment:
Does it "support" fire and forget? I think the class just "is" fire and forget, though the app can keep hold of it for status updates?